motleycrew/tools/aider_tool.py

```diff
@@ -0,0 +1,49 @@
+from motleycrew.common.utils import ensure_module_is_installed
+
+try:
+    from aider.coders import Coder
+    from aider.models import Model
+except ImportError:
+    ensure_module_is_installed("aider-chat", "poetry run pip install aider-chat")
```

Comment:
why poetry here?

Reply:
also, let's put this inside AiderTool init in case we want to import this module in some __init__.py